cypress/integration/channel/add_users_to_channel_spec.js

```diff
@@ -0,0 +1,102 @@
+            cy.get(`#post_${id}`).find('.mention-link').first().click({force: true}).then(($el) => {
+                // * Then the user's profile pop-over should be displayed when their username is clicked, and the pop-over is fully visible
+                cy.get('#user-profile-popover').should('be.visible');
+
+                // * The pop-over title  at the top of the pop-over should be the same as the username link
+                const userName = $el[0].innerHTML;
+                const escapedUsername = userName.replace('@', '').replace('.', '-');
+                cy.get(`#userPopoverUserName-${escapedUsername}`).should('contain', userName);
+            });
+        });
+
+        // # Select 'sint', another channel on the left hand side
+        cy.get('#sidebarItem_suscipit-4').click();
+
+        // # Click 'sint' at the top of the center pane to access the drop-down menu
+        cy.get('#channelHeaderTitle').click();
+
+        // * The dropdown menu of the 'sint' channel header should be visible;
+        cy.get('#channelHeaderDropdownMenu').should('be.visible');
+
+        // # Click 'Add Members'
+        cy.get('#channelAddMembers').click({});
+
+        // * Assert that modal appears
+        cy.get('#addUsersToChannelModal').should('be.visible');
+
+        // # Click 3 times the first row clickable of the modal to select 3 different users
+        cy.get('#userAddList').first().click();
+        cy.get('#userAddList').first().click();
+        cy.get('#userAddList').first().click();
+
+        // # Click the button "Add" to add these 3 users to 'sint' channel
+        cy.get('#saveItems').click();
+
+        // # Wait 2 seconds to allow the popup to disappear
+        cy.wait(2000); // eslint-disable-line
+
+        cy.getLastPostId().then((id) => {
+            // # On the system message advising users were added to the channel (combined) that includes a link such as "and 2 others", click to expand the usernames
+            cy.get(`#post_${id}`).find('.markdown__paragraph-inline').siblings('a').first().click().then(() => {
+                cy.get(`#post_${id}`).find('.markdown__paragraph-inline').first().find('.mention-link').each(($el) => {
+                    // # Then click each username link
+                    cy.wrap($el).click();
+
+                    // * The profile pop-over of each user should be displayed when their username is clicked, and the pop-over is fully visible
+                    const userName = $el[0].innerHTML;
+                    cy.get('#user-profile-popover').should('be.visible');
+
+                    // * The pop-over title  at the top of the pop-over should be the same as the username link for each user
+                    const escapedUsername = userName.replace('@', '').replace('.', '-');
+                    cy.get(`#userPopoverUserName-${escapedUsername}`).should('contain', userName);
+                });
+            });
```

Comment:
If above is applied, then this can be simplified as,
```javascript
// # System message should contain message that users were added to the channel
cy.get(`#postMessageText_${id}`).should('contain', '2 others were added to the channel by you');

// # Click "2 others" to expand more users
cy.get(`#post_${id}`).find('.markdown__paragraph-inline').siblings('a').first().click();

// * Verify each username link
verifyMentionedUserAndProfilePopover(id);
```